.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Mocha Tests",
+            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+            "args": [
+                "--timeout",
+                "999999",
+                "--colors",
+                "${workspaceFolder}/test"
+            ],
+            "internalConsoleOptions": "openOnSessionStart"
+        }
+    ]
```

Comment:
EOF

Reply:
What is the meaning of EOF? I need to delete the ".vscode" dir?

Reply:
Just add an enter at the end of the file

Reply:
ok, thanks.